packages/contentstack-config/src/utils/interactive.ts

```diff
@@ -96,3 +96,23 @@ export async function askEarlyAccessHeaderAlias(): Promise<string> {
     validate: inquireRequireFieldValidation,
   });
 }
+
+export function handleErrorMsg(err) {
+  if (err?.errorMessage) {
+    cliux.print(`Error: ${err.errorMessage}`, { color: 'red' });
+  } else if (err?.message) {
+    cliux.print(`Error: ${err.message}`, { color: 'red' });
+  } else {
+    console.log(err);
+    cliux.print(`Error: ${messageHandler.parse('CLI_CONFIG_API_FAILED')}`, { color: 'red' });
+  }
+  process.exit(1);
+}
```

Comment:
@harshithad0703 , don't think it's required.

Reply:
removed this as its not required